### PR TITLE
Add company profile segment to user guide

### LINKED ISSUE #67: Update Company Profiles feature in the User Guide for v1.1

### PR DESCRIPTION
**Pull request by shawn-nyk**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -83,6 +83,46 @@ Profile | me | category, descriptors
...
+- `add -com n/Garena i/Gaming j/Game developer`
+- `add -com n/Google i/Internet i/Cloud computing i/Artificial intelligence j/Software engineer j/Cloud architect`
+
+#### Deleting a company profile: `delete -com`
+
+Deletes a company profile from your list of company profiles.
+
+Format: `delete -com INDEX`
+
+Examples:
```
Comment (seanjyjy) [inline docs/UserGuide.md]: small issue maybe it should be example instead of examples?

File: docs/UserGuide.md [hunk 2]
```diff
@@ -83,6 +83,46 @@ Profile | me | category, descriptors
...
+- `edit -com 1 j/Game developer j/Software tester`
+- `edit -com 2 n/Google LLC i/Cloud computing i/Artificial intelligence i/Deep learning`
+
+#### Viewing a company profile: `view -com`
+
+Selects a company profile in your list of company profiles to show in detail.
+
+Format: `view -com INDEX`
+
+Examples:
```
Comment (seanjyjy) [inline docs/UserGuide.md]: same issue here

File: docs/UserGuide.md [hunk 3]
```diff
@@ -83,6 +83,46 @@ Profile | me | category, descriptors
...
+Format: `delete -com INDEX`
+
+Examples:
+- `delete -com 5`
+
+#### Editing a company profile: `edit -com`
+
+Edits a company profile from your list of company profiles.
+
+Format: `edit -com INDEX [n/COMPANY_NAME] [i/INDUSTRY]... [j/JOBS]...`
```
Comment (seanjyjy) [inline docs/UserGuide.md]: there seem to be some inconsistency here. do you mean JOB_TITLES instead?

File: docs/UserGuide.md [hunk 4]
```diff
@@ -233,15 +273,15 @@ Format: `exit`
 
 Type            | Action     | Format
 ----------------|------------|------------------
-**Company**     | **Add**    | `add -com n/COMPANY_NAME i/INDUSTRY j/JOBS`
+**Company**     | **Add**    | `add -com n/COMPANY_NAME i/INDUSTRY... j/JOB_TITLES...`
 &nbsp;          | **Delete** | `delete -com INDEX`
-&nbsp;          | **Edit**   | `edit -com INDEX [n/COMPANY_NAME] [i/INDUSTRY] [j/JOBS]`
+&nbsp;          | **Edit**   | `edit -com INDEX [n/COMPANY_NAME] [i/INDUSTRY]... [j/JOBS]...`
```
Comment (seanjyjy) [inline docs/UserGuide.md]: do you mean job_title

**Review by seanjyjy — COMMENTED** [review]
Overall looks great. Good job. Just a few nits and it will be good to merge.

**Comment by shawn-nyk** [comment]
> Overall looks great. Good job. Just a few nits and it will be good to merge.

Thanks for pointing them out Sean! Have gone ahead to fix those inconsistencies across the whole doc.

**Comment by seanjyjy** [comment]
LGTM

**Pull request merged**